### DISCUSSION TITLE
Any chance of reverting to the previous icon?

### DISCUSSION THREAD

**Discussion by kghandi** [aonez/Keka#1082] [Q&A] [answered]
I loved the old rolly-polly icon. It was cute and distinctive, and was very easy to pick out in the Launchpad. The new icon has a couple of issues. For one, unless you are familiar with the previous rolly-polly icon, you would have a hard time figuring out what it represents. The new icon colors have a petty low contrast, which makes it even more difficult to determine what the image represents. But most importantly, it is actually pretty hard to pick out the icon among other app icons since it looks like a fuzzy brown square. I appreciate the effort that was put into updating the icon, but I think the old icon was better.

**Comment by gingerbeardman** [comment, marked as answer]
See here for how you can do this yourself: https://github.com/aonez/Keka/issues/988#issuecomment-982536993

@aonez I think this info should be added to the Wiki. I do not have permission to add pages.

**Reply by kghandi** [reply]
Thank you for the link. That did the trick.
